src/media/js/addon/containers/__tests__/submit.test.js

```diff
@@ -14,7 +14,7 @@ describe('AddonSubmit', () => {
                           {...props}/>
     );
     assert.equal(ReactDOMHelper.queryTagAll(component, 'form').length, 1);
-    assert.notOk(ReactDOMHelper.queryTag(component, 'button').props.disabled);
+    assert.ok(ReactDOMHelper.queryTag(component, 'button').props.disabled);
```

Comment:
Swapped this assertion because the button is now disabled until you've uploaded an addon that passes validation.